scripts/validate.js

```diff
@@ -1,38 +1,67 @@
 /* global hexo */
 
-'use strict';
+'use strict'
 
-const { join } = require('path');
+const { join } = require('path')
 
-const { listDir } = require('hexo-fs');
-const { log } = hexo;
-const sizeOf = require('image-size');
+const { listDir } = require('hexo-fs')
+const { log } = hexo
+const sizeOf = require('image-size')
+
+const yaml = require('js-yaml')
+const fs = require('fs')
+
+async function validateThemeNames() {
+  let isValidationPassed = true
+  const themeDataFile = fs.readFileSync(hexo.source_dir + '_data/themes.yml')
+  const themeData = yaml.load(themeDataFile)
```

Comment:
Use `const themeData = hexo.locals.get('data').themes`

Reply:
Changes are done, thanks for the suggestion.